src/main/scala/xerial/sbt/Sonatype.scala

```diff
@@ -51,10 +52,14 @@ object Sonatype extends AutoPlugin {
     sonatypeProfileName := organization.value,
     sonatypeRepository := "https://oss.sonatype.org/service/local",
     sonatypeCredentialHost := "oss.sonatype.org",
+    sonatypeProjectHosting := None,
     publishMavenStyle := true,
     pomIncludeRepository := { _ =>
       false
     },
+    homepage := sonatypeProjectHosting.value.map(h => url(h.homepage)).orElse(homepage.value),
+    scmInfo := sonatypeProjectHosting.value.map(_.scmInfo).orElse(scmInfo.value),
+    developers := sonatypeProjectHosting.value.map(_.developers).orElse(Some(developers.value)).getOrElse(List.empty),
```

Comment:
To allow users add their own developer list to sonatypeProjectHosting, we should not set the default value `List(developer)` in ProjectHosting class.

Reply:
What if we swap the order of precedence here?

So if `developers` is a list with more than one element we use that otherwise we derive it from the ProjectHosting instance. That way the users can override this setting quite easily.

(This could be done to the other settings as well.)